src/v2/guide/class-and-style.md

```diff
@@ -1,32 +1,32 @@
 ---
-title: Class and Style Bindings
+title: Enlace de Clases y Estilos
```

Comment:
Please review **all titles**, since we are following the MDN guide:
> Cuando escribas los nombres de las páginas y de los encabezados, recuerda que sólo debe empezar con mayúscula la primera palabra de la frase.